docs/cicd/cicd-metrics.md

```diff
@@ -0,0 +1,548 @@
+linkTitle: CICD metrics
+--->
+
+# Semantic Conventions for CICD Metrics
+
+**Status**: [Experimental][DocumentStatus]
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [VCS Metrics](#vcs-metrics)
+  - [Metric: `vcs.repository.change.count`](#metric-vcsrepositorychangecount)
+  - [Metric: `vcs.repository.change.time_open`](#metric-vcsrepositorychangetime_open)
+  - [Metric: `vcs.repository.change.time_to_approval`](#metric-vcsrepositorychangetime_to_approval)
+  - [Metric: `vcs.repository.change.time_to_merge`](#metric-vcsrepositorychangetime_to_merge)
+  - [Metric: `vcs.repository.count`](#metric-vcsrepositorycount)
+  - [Metric: `vcs.repository.ref.count`](#metric-vcsrepositoryrefcount)
+  - [Metric: `vcs.repository.ref.lines_added`](#metric-vcsrepositoryreflines_added)
+  - [Metric: `vcs.repository.ref.lines_deleted`](#metric-vcsrepositoryreflines_deleted)
+  - [Metric: `vcs.repository.ref.revisions_ahead`](#metric-vcsrepositoryrefrevisions_ahead)
+  - [Metric: `vcs.repository.ref.revisions_behind`](#metric-vcsrepositoryrefrevisions_behind)
+  - [Metric: `vcs.repository.ref.time`](#metric-vcsrepositoryreftime)
+  - [Metric: `vcs.repository.contributor.count`](#metric-vcsrepositorycontributorcount)
+
+<!-- tocstop -->
+
+## VCS Metrics
+
+The conventions described in this section are specific to Version Control Systems.
+
+**Disclaimer:** These are initial VCS metrics and attributes
+but more may be added in the future.
+
+### Metric: `vcs.repository.change.count`
+
+<!-- semconv metric.vcs.repository.change.count(metric_table) -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+| Name     | Instrument Type | Unit (UCUM) | Description    | Stability |
+| -------- | --------------- | ----------- | -------------- | --------- |
+| `vcs.repository.change.count` | Gauge | `{change}` | The number of changes (pull requests) in a repository, categorized by their state (either open or merged) | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+<!-- END AUTOGENERATED TEXT -->
+<!-- endsemconv -->
+
+<!-- semconv metric.vcs.repository.change.count -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`vcs.change.state`](/docs/attributes-registry/vcs.md) | string | The state of a change (pull request). | `open`; `merged` | `Required` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
Contrary to https://github.com/adrielp/opentelemetry-collector-contrib/blob/903eec5382352e4d663fadd2dc7f1cc1e37f62fd/receiver/githubreceiver/documentation.md all attributes have the `vcs.` prefix to align with existing vcs attribute definitions.
Anything to be changed?